command/server.go

```diff
@@ -871,6 +871,90 @@ func (c *ServerCommand) setupStorage(config *server.Config) (physical.Backend, e
 	return backend, nil
 }
 
+func (c *ServerCommand) InitListeners(config *server.Config, disableClustering bool, infoKeys *[]string, info *map[string]string) (int, error, []listenerutil.Listener, []*net.TCPAddr) {
```

Comment:
is the first return parameter a return code? it might be helpful to indicate the intent there

Reply:
Done!